c/custodia/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:xenial
+MAINTAINER "Eshant Gupta <guptaeshant@gmail.com>"
+ENV TOXENV=py36-noextras
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+RUN \
+    # Install dependencies
+    apt-get update && \
```

Comment:
we should use the "-y" flag for apt-get update as well as otherwise it can cause the script execution to wait on user input in certain scenarios